<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>762</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Plot parameters</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinAndMaxSize</enum>
         </property>
         <item>
          <layout class="QFormLayout" name="formLayout_2">
           <property name="sizeConstraint">
            <enum>QLayout::SetMinAndMaxSize</enum>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="layer_label">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Layer</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QgsMapLayerComboBox" name="layer_selection">
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="chart_type_label">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Chart type</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="chart_selection">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <item>
              <property name="text">
               <string>Scatter plot</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Line plot</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Bar plot</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Box plot</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Histogram</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>3D scatter plot</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Parallel coordinates</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Correlation matrix</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="x_label">
             <property name="text">
              <string>X</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QgsFieldComboBox" name="x_data_selection">
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="y_label">
             <property name="text">
              <string>Y</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QgsFieldComboBox" name="y_data_selection">
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="label_17">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>290</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Plot style</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QFormLayout" name="formLayout_4">
           <item row="0" column="0">
            <widget class="QLabel" name="color_label">
             <property name="text">
              <string>Color</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QgsColorButton" name="color_selection"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="marker_type_label">
             <property name="text">
              <string>Marker type</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="marker_type_selection"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="lineTypeLabel">
             <property name="text">
              <string>Line type</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="line_type_selection"/>
           </item>
           <item row="3" column="1">
            <widget class="QgsOpacityWidget" name="opacity_selection">
             <property name="opacity">
              <double>0.600000000000000</double>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Opacity</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="plot_button">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Plot</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="clear_button">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Clear plots</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>30</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QWidget" name="plot_container" native="true">
       <property name="minimumSize">
        <size>
         <width>450</width>
         <height>0</height>
        </size>
       </property>
       <widget class="QWidget" name="layoutWidget_3">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>150</y>
          <width>197</width>
          <height>203</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="6" column="0">
          <widget class="QLabel" name="label_40">
           <property name="text">
            <string>50% quantile</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_46">
           <property name="text">
            <string>Mean</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="label_49">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QLabel" name="label_43">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_50">
           <property name="text">
            <string>Max</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_38">
           <property name="text">
            <string>25% quantile</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLabel" name="label_41">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="label_55">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="label_51">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_47">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_52">
           <property name="text">
            <string>Standard deviation</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_54">
           <property name="text">
            <string>Relative std. deviation</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_48">
           <property name="text">
            <string>Min</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="label_53">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLabel" name="label_39">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_42">
           <property name="text">
            <string>75% quantile</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_44">
           <property name="text">
            <string>Skew</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QLabel" name="label_45">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="label_37">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>120</y>
          <width>151</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Summary statistics</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsColorButton</class>
   <extends>QToolButton</extends>
   <header>qgscolorbutton.h</header>
  </customwidget>
  <customwidget>
   <class>QgsFieldComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsfieldcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsmaplayercombobox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsOpacityWidget</class>
   <extends>QWidget</extends>
   <header>qgsopacitywidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
